templates/vue/src/components/TapestryApp.vue

```diff
@@ -1,6 +1,19 @@
 <template>
   <div id="app-container" :class="{ 'sidebar-open': isSidebarOpen }">
-    <toolbar />
+    <toolbar style="margin-bottom: 100px;" />
+    <b-modal id="avatar-modal" size="xl" :visible="!hasAvatar && avatarsEnabled">
+      <b-container class="avatar-container">
+        <avatar-form ref="AvatarForm" />
+      </b-container>
+      <template slot="modal-footer">
+        <b-button size="sm" variant="secondary" @click="cancel">
+          Cancel
+        </b-button>
+        <b-button size="sm" variant="primary" @click="saveAvatar">
+          Save Avatar
+        </b-button>
+      </template>
+    </b-modal>
```

Comment:
Is this needed here? I thought we have the modal inside of UserSettingModal which is inside the Toolbar component already.